share/random_namespace.go

```diff
@@ -38,22 +41,53 @@ func RandomBlobNamespace() Namespace {
 	for {
 		id := RandomBlobNamespaceID()
 		namespace := MustNewV0Namespace(id)
-		if isBlobNamespace(namespace) {
+		if namespace.IsBlobNamespace() {
 			return namespace
 		}
 	}
 }
 
-// isBlobNamespace returns an true if this namespace is a valid user-specifiable
-// blob namespace.
-func isBlobNamespace(ns Namespace) bool {
-	if ns.IsReserved() {
-		return false
+// AddInt adds arbitrary int value to namespace, treating namespace as big-endian
+// implementation of int. Note: should only be used for tests.
+func AddInt(t *testing.T, n Namespace, val int) Namespace {
```

Comment:
Why limit they use by tests only? It used to just return an error in node code

Reply:
I thought we need this function for testing purposes only. If not, could you please share more context

Reply:
It is just a handlful function as namespace method and might be used by some users. Making it free func makes it much harder to find and generally does not allign with out package structure, where we make such functions as method on core type (namespace). I see it handful to be it as part of common API, even if it is not currently used by us.

Reply:
So, your proposal is to make it a method of Namespace structure, right?

Reply:
moved to methods